openstack/billing/templates/prometheus-alerts.yaml

```
@@ -8,7 +8,7 @@ metadata:
     type: alerting-rules
     prometheus: openstack
 
-{{- $keystone_namespace := .Values.is_global | ternary "monsoon3global" "monsoon3" }}
+{{- $keystone_namespace := eq .Values.global.is_global_region true | ternary "monsoon3global" "monsoon3" }}
```

`is_global_region` is already a bool, so we don't need to `eq/ne` against `true` here and in several other places:

```suggestion
{{- $keystone_namespace := .Values.global.is_global_region | ternary "monsoon3global" "monsoon3" }}
```

For this to be well-behaved, `values.yaml` should declare a default value:

```yaml
global:
  is_global_region: false
```

---

Ah, I did not go the "default"-way and rather assumed the flag can be undefined in the values. Will change that, thanks!